WindowManager/app/src/main/java/com/example/windowmanagersample/SplitLayout.kt

```diff
@@ -195,7 +193,6 @@ class SplitLayout : FrameLayout {
 
     private fun isValidFoldFeature(displayFeature: DisplayFeature): Boolean {
         val feature = displayFeature as? FoldingFeature ?: return false
-        return (feature.type == TYPE_FOLD || feature.type == TYPE_HINGE) &&
-            getFeaturePositionInViewRect(feature, this) != null
+        return getFeaturePositionInViewRect(feature, this) != null
```

Comment:
I would simplify this using `let`

```suggestion
private fun isValidFoldFeature(displayFeature: DisplayFeature) =
        (displayFeature as? FoldingFeature)?.let { getFeaturePositionInViewRect(it, this) } ?: false
```

Reply:
Thanks for the suggestion!

I've pushed an update with a slightly modified version of what you suggested. PTAL